spec/controllers/spotlight/solr_controller_spec.rb

```diff
@@ -37,6 +37,18 @@
         expect(doc).to include 'a' => 1
       end
 
+      context "when the index is not writable" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.